lib/metasploit/framework/ntds/account.rb

```diff
@@ -0,0 +1,178 @@
+module Metasploit
+  module Framework
+    module NTDS
+      # This class represents an NTDS account structure as sent back by Meterpreter's
+      # priv extension.
+      class Account
+
+        # @!attribute description
+        #   @return [String] The AD Account Description
+        attr_accessor :description
+        # @!attribute disabled
+        #   @return [TrueClass] If the AD account is disabled
```

Comment:
YARD recognizes `@return [Boolean]` or `@return [true]` and `@return [false]`.  You don't need to use the boolean class names.